content/lang/de/about/browser-support.md

```diff
@@ -5,7 +5,7 @@ permalink: '/about/browser-support'
 
 # Browserunterstützung
 
-Prect untersützt moderne Browser (Chrome, Firefox, Safari, Edge) und IE9+. Es sollte in IE7 sowie IE8 funktionieren, setzt allerdings einige Polyfiller voraus. Wenn es beabsichtigt ist, Preact zu benutzen und diese älteren Browser zu untersützen, wird empfohlen, einen Polyfiller wie zum Beispiel [es5-shim] oder [babel-polyfill] zu verwenden.
+Preact untersützt moderne Browser (Chrome, Firefox, Safari, Edge) und IE9+. Es sollte in IE7 sowie IE8 funktionieren, setzt allerdings einige Polyfiller voraus. Wenn es beabsichtigt ist, Preact zu benutzen und diese älteren Browser zu unterstützen, wird empfohlen, einen Polyfiller wie zum Beispiel [es5-shim] oder [babel-polyfill] zu verwenden.
```

Comment:
Native german here:

Reply:
Preact unterstützt Chrome, Firefox, Safari, Edge sowie IE9+ vollständig und sollte auch in IE7 sowie IE8 weitgehend funktionieren, wobei dann einige Polyfills(link?) vorausgesetzt werden.